code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -677,3 +677,6 @@ It'll return null if the organ doesn't correspond, so include null checks when u
 
 	if(H.see_override)	//Override all
 		H.see_invisible = H.see_override
+
+/datum/species/proc/water_act(mob/living/carbon/human/M, volume, temperature, source)
+	..()
```

Comment:
There....is no parent; this is the parent.

This should encapsulate human/water_act()

```
 	if(temperature >= 330)	bodytemperature = bodytemperature + (temperature - bodytemperature)
 	if(temperature <= 280)	bodytemperature = bodytemperature - (bodytemperature - temperature)
```
at the base level, then have the Grey's water act be like it is.